### DISCUSSION TITLE
Unable to deploy managed private endpoints

### DISCUSSION THREAD

**Discussion by waldemar-sarmiento** [Azure-Player/azure.datafactory.tools#307] [Show and tell]
I don't think this is a bug but I cannot find any documentation on how to do it and I am getting an error:

**##[error]ADFT0010: Wrong path defined in config for object(path): managedPrivateEndpoint.pep-adf-dev-syn-dev(properties.name)**

**To Reproduce**
In order to deploy the managed private endpoints that go into Synapse (adf->synapse with private access)
I change it in the config file like this:

type,name,path,value
managedPrivateEndpoint,"pep-adf-dev-syn-dev-sql","$.name", "pep-adf-uat-syn-uat-sql" 
managedPrivateEndpoint,"pep-adf-dev-syn-dev-sql","$.properties", "{'privateLinkResourceId': '/subscriptions/guid/resourceGroups/rg-uat-001/providers/Microsoft.Synapse/workspaces/syn-uat-001','groupId': 'sql','fqdns': ['syn-uat.sql.azuresynapse.net']}"

**Comment by NowinskiK** [comment]
First error: You can't change **name** of the object.
This discussion can help: https://github.com/Azure-Player/azure.datafactory.tools/discussions/298 in terms of second point.

**Comment by waldemar-sarmiento** [comment]
Thank you! That helped! 
Fixed it in an instant:

managedPrivateEndpoint,"pep-adf-dev-syn-dev-sql",$.properties.privateLinkResourceId, "/subscriptions/guid/resourceGroups/rg-uat-001/providers/Microsoft.Synapse/workspaces/syn-uat-001"
managedPrivateEndpoint,"pep-adf-dev-syn-dev-sql",$.properties.groupId, sql
managedPrivateEndpoint,"pep-adf-dev-syn-dev-sql",$.properties.fqdns[0], "syn-uat.sql.azuresynapse.net"

Cannot change the endpoint name but will make it not dependent of the environment.
